build-tests/api-extractor-scenarios/etc/test-outputs/apiItemKinds/api-extractor-scenarios.api.json

```diff
@@ -685,6 +685,52 @@
               },
               "isStatic": false
             },
+            {
+              "kind": "Property",
+              "canonicalReference": "api-extractor-scenarios!SimpleClass#someReadonlyProp:member",
+              "docComment": "",
+              "excerptTokens": [
+                {
+                  "kind": "Content",
+                  "text": "readonly someReadonlyProp = 5;"
```

Comment:
In general API Extractor should handle these `readonly` properties better (we should be able to surface the "type" as `5`), but that's a separate FR from this fix.